src/main.cpp

```diff
@@ -138,6 +138,7 @@ void usage()
   std::cerr << "    BPFTRACE_MAX_TYPE_RES_ITERATIONS  [default: 0] number of levels of nested field accesses for tracepoint args" << std::endl;
   std::cerr << "    BPFTRACE_PERF_RB_PAGES            [default: 64] pages per CPU to allocate for ring buffer" << std::endl;
   std::cerr << "    BPFTRACE_STACK_MODE               [default: bpftrace] Output format for ustack and kstack builtins" << std::endl;
+  std::cerr << "    BPFTRACE_STACK_THRESHOLD          [default: 32] max bytes to store objects on BPF stack vs. global variable" << std::endl;
```

Comment:
```suggestion
  std::cerr << "    BPFTRACE_STACK_THRESHOLD          [default: 32] max bytes of individual objects to store them on the BPF stack vs. a global variable" << std::endl;
```

Reply:
"global variable" kinda leaks an unnecessary implementation detail. Maybe say "preallocated memory"?

Reply:
nit: "stack threshold" seems backwards (implying a minimum for the stack). maybe "offstack threshold"?

Reply:
I think maybe both are ambiguous. How about "on_stack_limit" or "on_stack_max" or "off_stack_min"

Reply:
Lets discuss naming offline

Reply:
on_stack_limit seems to be the choice, lets go with that